Sources/SmithyHTTPAPI/Headers.swift

```diff
@@ -5,7 +5,7 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
-public struct Headers {
+public struct Headers: Sendable {
```

Comment:
`Headers` and `Header` may be marked `Sendable` without any further change since they are `struct`s that only store value types.